lib/ini/ini.nit

```diff
@@ -66,7 +66,7 @@ class ConfigTree
 	fun at(key: String): nullable Map[String, String] do
 		var node = get_node(key)
 		if node == null then return null
-		var map = new HashMap[String, String]
+		var map = new ArrayMap[String, String]
```

Comment:
`HashMap` already keeps the order if I remember correctly. Are you sure the order is not preserved?